index.html

```diff
@@ -404,3 +404,5 @@ <h1 class="contactus-heading text-color-1 no-margin-bottom">I'm always intereste
 </body>
 
 </html>
+
```

Comment:
- [ ] The text headings should have a meaning hierarchy. In other words, there should be one `h1` tag on a web page and the rest can be `h2-h6` tags. Currently, you have several `h1` tags in your `index.html` file. Kindly implement this change. You can read more on accessibility guidelines for text headings in this [article](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). 

![image](https://user-images.githubusercontent.com/77106962/223842439-86750719-e9db-42d3-aeb6-5482d66ae0eb.png)